torch/autograd/function.py

```diff
@@ -502,10 +501,11 @@ def __call__(self, *args, **kwargs):
 
     @staticmethod
     def vmap(info, in_dims, *args):
-        r"""Defines a rule for the behavior of this autograd.Function underneath
-        :func:`torch.vmap`. For a :func:`torch.autograd.Function` to support
-        :func:`torch.vmap`, you must either override this staticmethod, or set
-        ``generate_vmap_rule`` to ``True`` (you may not do both).
+        r"""Define vmap behavior for autograd functions, override or set "generate_vmap_rule" to enable "torch.vmap".
```

Comment:
it should be possible to use :func:`torch.vmap` and :func:`torch.vmap`. Please change them back everywhere in the file.
Also, we use double backticks rather than quotes in values like ``True`` - this syntax should be supported by pydocstyle.

Reply:
I have changed them back everywhere. Please review it.